packages/sdks-tests/src/specs/index.ts

```diff
@@ -93,6 +93,7 @@ import { DYNAMIC_BUTTON } from './dynamic-button.js';
 import { SYMBOLS_WITH_LIST_CONTENT_INPUT } from './symbols-with-list-content-input.js';
 import { COLUMNS_VERTICAL_CENTERING } from './columns-vertical-centering.js';
 import { SECTION_CHILDREN } from './section-children.js';
+import { MAIN_CONTENT } from './get-content-symbol-update-entry.js';
```

Comment:
we can follow the naming convention, keep this same as the route

```suggestion
import { MAIN_CONTENT as SYMBOL_UPDATE_ENTRIES } from './get-content-symbol-update-entry.js';
```